invokeai/app/api_app.py

```diff
@@ -146,15 +130,23 @@ def overridden_redoc():
 
 
 def invoke_api():
+    # parse command-line settings, environment and the init file
+    # (this is a module global)
+    global web_config
+    web_config = get_invokeai_config(InvokeAIWebConfig)
+    app.add_middleware(
+        CORSMiddleware,
+        allow_origins=web_config.allow_origins,
+        allow_credentials=web_config.allow_credentials,
+        allow_methods=web_config.allow_methods,
+        allow_headers=web_config.allow_headers,
+    )
```

Comment:
[This suggested change](https://github.com/invoke-ai/InvokeAI/pull/3311/files#diff-0aa99a16529b3476fb4e8f83ff96496dace296060c35544058cbebbfa351a5dc) allows for launching the API both with the `invokeai-node-web` script *or* using the `uvicorn` ASGI server directly, enabling use of a debugger and hot-reload in development. Could we do it that way, or does that break anything I'm not aware of?

Reply:
This looks good to me. I will add.

Reply:
I've implemented the suggested change to web_config.